tracee-ebpf/tracee/consts.go

```diff
@@ -168,6 +168,8 @@ const (
 	SecurityBPFEventID
 	SecurityBPFMapEventID
 	SecurityKernelReadFileEventID
+	SecurityPathMknodEventID
```

Comment:
This event is defined under CONFIG_SECURITY_PATH (https://elixir.bootlin.com/linux/v5.12.19/source/security/security.c#L1082) and will not work on all distros.
If you add SecurityInodeMknodEventID - no need for this one

Reply:
your'e right, i missed it. i'll remove this event.